### PR TITLE
feat: adding a link to open the section in seb

### PR DESCRIPTION
**Pull request by felipemontoya**
This PR adds a link in the blocked page so that learners are invited to open the section again in SEB

### CODE REVIEW

File: seb_openedx/middleware.py [hunk 1]
```diff
@@ -137,6 +139,17 @@ def handle_access_denied(self, request, view_func, view_args, view_kwargs, cours
         is_banned, new_ban = ban_user(user_name, course_key, request.user.username)
         is_courseware_view = bool(view_func.__name__ == get_courseware_index_view().__name__)
         context.update({"banned": is_banned, "is_new_ban": new_ban})
+
+        if not is_banned:
+            try:
+                usage_key_string = request.resolver_match.kwargs.get('usage_key_string')
+                url = reverse('jump_to', kwargs=dict(course_id=str(course_key), location=str(usage_key_string)))
```
Comment (BryanttV) [inline seb_openedx/middleware.py:146]: Could we use a dictionary directly instead of calling `dict()`?

File: seb_openedx/middleware.py [hunk 2]
```diff
@@ -137,6 +139,17 @@ def handle_access_denied(self, request, view_func, view_args, view_kwargs, cours
...
+                parsed_url = urllib.parse.urlparse(url)
+                seb_url = parsed_url._replace(scheme="seb", netloc=request.get_host())
+                context.update({"jump_to_link": urllib.parse.urlunparse(seb_url)})
+            except Exception as e:
```
Comment (BryanttV) [inline seb_openedx/middleware.py:150]: Is it possible to catch more specific exception/exceptions?

**Review by BryanttV — APPROVED** [review]

File: seb_openedx/middleware.py (hunk 2 above)
Comment (felipemontoya) [reply seb_openedx/middleware.py:150]: I did not want to risk a 500 error for this. So it either renders a good link or nothing. urllib has a very stable api so the errors may only come from the info we pass to it.

File: seb_openedx/middleware.py (hunk 1 above)
Comment (felipemontoya) [reply seb_openedx/middleware.py:146]: I took some code from the dashboard page as the inspiration. I have no particular inclination for or against dict

Comment (BryanttV) [reply seb_openedx/middleware.py:146]: The Pylint extension in VSCode suggested I to change it, here is the reference: https://pylint.readthedocs.io/en/latest/user_guide/messages/refactor/use-dict-literal.html

**Pull request merged**